kitchen.dokken.yml

```diff
@@ -0,0 +1,94 @@
+---
+driver:
+  name: dokken
+  privileged: true
+  chef_version: <%= ENV['CHEF_VERSION'] || 'current' %>
+  volumes: ['/mnt']
+
+transport:
+  name: dokken
+
+provisioner:
+  name: dokken
+  deprecations_as_errors: true
+
+verifier:
+  name: inspec
+
+platforms:
+  - name: amazonlinux
+    driver:
+      image: dokken/amazonlinux
+      privileged: true
+      pid_one_command: /sbin/init
+      intermediate_instructions:
+        - RUN export CHEF_LICENSE="accept"
```

Comment:
We should get this in the orb before we call Test Kitchen so we don't have to add it eveyrwhere.

Reply:
Yeah it's actually in the orb now. I just didn't tidy this up